src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Net
     //
     // A list of cookies maintained in Sorted order. Only one cookie with matching Name/Domain/Path
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, PublicKeyToken=b77a5c561934e089")]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Why are we adding "Culture=neutral" attributes?  Is this different from .NET Framework?